src/components/Page.tsx

```diff
@@ -63,6 +67,8 @@ const Page: FunctionComponent<PageProps & HTMLAttributes<HTMLDivElement>> = ({
             data-title={title}
             data-backbutton={`${isBackButtonEnabled}`}
             data-menubutton={`${isMenuButtonEnabled}`}
+            data-dom-cache={`${isDomCacheEnabled}`}
+            data-backdroptype={`${backDropType}`}
```

Comment:
This doesn't need to be wrapped in a string since it is already a string.

Reply:
@thornbill updated https://github.com/jellyfin/jellyfin-web/pull/3567/commits/76eff104d8a40e66a32ba07e9cc6b0f75ab16d6e